app/controllers/public/sessions.js

```diff
@@ -2,13 +2,17 @@ import classic from 'ember-classic-decorator';
 import { computed } from '@ember/object';
 import Controller from '@ember/controller';
 import moment from 'moment';
+import { timezones } from 'open-event-frontend/utils/dictionary/date-time';
 
 @classic
 export default class SessionsController extends Controller {
+
   queryParams = ['sort'];
   sort = 'starts-at';
   isTrackVisible = false;
-
+  timezones = timezones;
+  currentTimezone = moment.tz.guess();
```

Comment:
This should be event timezone by default, not local timezone

Reply:
Session page drop down is showing event timezone by default, not local timezone

Reply:
> currentTimezone = moment.tz.guess();

this line is for in other condition current timezone, other than event timezone